shaman.gemspec

```diff
@@ -13,18 +15,16 @@ Gem::Specification.new do |spec|
   spec.homepage      = 'https://github.com/infinum/shaman'
   spec.license       = 'MIT'
 
+  spec.required_ruby_version = '>= 2.6'
```

Comment:
Do we have a concrete blocker why we can't go lower? I'm not sure what the mobile build pipelines are actually running 🤔

Reply:
I set this based on best judgment and limited analysis (considering only runtime dependencies). Not that I think about it, I think we should be strict and support only Ruby version > 3. It was released more than 3 years ago, and 2.X reached EOL more than a year ago. Supporting older Ruby versions will also limit our ability to upgrade to newer dependency versions.

Reply:
This would be a breaking change right? In which release would we package this, the upcoming majore one?

Reply:
It's a breaking change and it'll require a major release.